src/Parser/MarksmanshipHunter/Modules/FocusChart/Focus.js

```diff
@@ -0,0 +1,374 @@
+//Based on Main/Mana.js and Parser/VengeanceDemonHunter/Modules/PainChart
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChartistGraph from 'react-chartist';
```

Comment:
This isn't something that we made clear in any way and you don't have to fix it (at all), but we moved to use ChartJS instead of Chartist and intended for all new charts to be built with that instead. The status charts are an example of this. Since the only functional gain for migrating this for you would be tooltips you're in no way required to change this (but if you're motivated to do so it would be greatly appreciated! ;)).

Reply:
I'll make a note of it, and I'll make sure that I update it to chartJS sometime very soon.